### DISCUSSION TITLE
[firebase_dynamic_links] iOS :  CocoaPods could not find compatible versions for pod

### DISCUSSION THREAD

**Discussion by callmejm** [firebase/flutterfire#7144] [Q&A]
Hi, I facing iOS build issue
    
    firebase_core: ^1.7.0
    firebase_dynamic_links: ^2.0.10

    
    
    Analyzing dependencies
    firebase_analytics: Using Firebase SDK version '8.7.0' defined in 'firebase_core'
    firebase_core: Using Firebase SDK version '8.7.0' defined in 'firebase_core'
    firebase_dynamic_links: Using Firebase SDK version '8.7.0' defined in 'firebase_core'
    [!] CocoaPods could not find compatible versions for pod "firebase_dynamic_links":
      In Podfile:
        firebase_dynamic_links (from `.symlinks/plugins/firebase_dynamic_links/ios`)
    
    Specs satisfying the `firebase_dynamic_links (from `.symlinks/plugins/firebase_dynamic_links/ios`)` dependency were found, but they required a higher minimum deployment target.

**Comment by DataHearth** [comment]
Have any updates on it ? I have the same error...
